deployment/ansible/group_vars/production.example

```diff
@@ -0,0 +1,241 @@
+---
+## GLOBAL APP SETTINGS
+# Which version of the app to install. Specifically, this designates a tag on quay.io that should be
+# used when pulling docker images. A list of valid tags can be found here:
+# https://quay.io/repository/azavea/driver-app?tag=latest&tab=tags
+# Generally, when setting up a new instance, you should use the highest-number tag available. The
+# 'latest' tag should be avoided when setting up new instances for production usage, because it may
+# contain bugs.
+app_version: "0.0.0"
+# The URL at which users will access the application. Do NOT include 'http' or 'https'; all DRIVER
+# instances are configured to use https.
+app_domain_name: "your.driver-domain.here"
+
+# The *internal* IP address of the hosts for the different components of the web application. DRIVER
+# is currently divided up to run on three servers: A database, a web app server, and a Celery
+# (asynchronous tasks) server. If your servers have only a single IP address, then use that. If they
+# have multiple IP addresses, use the internal, non-public-facing IP address of the database server.
+database_server_ip: "11.11.11.11"
+app_server_ip: "22.22.22.22"
+celery_server_ip: "33.33.33.33"
+
+## POSTGRESQL SETTINGS
+# The username, password, and database name that the main web app will use when connecting to the
+# database server. These do not need to already exist on the database server; they will be added
```

Comment:
Is it fine if they do?

Reply:
Yup, should be, I'll add a note about that.